fili-core/src/main/java/com/yahoo/bard/webservice/druid/model/filter/SearchFilter.java

```diff
@@ -78,18 +95,26 @@ private SearchFilter(Dimension dimension, String type, String value) {
         return query;
     }
 
+    private String getQueryType() {
+        return query.get("type");
```

Comment:
`type` is used multiple times. It's more maintainable to have it as a static final String.